src/main/groovy/JavadocGroupBuilder.groovy

```diff
@@ -14,7 +14,7 @@ public class JavadocGroupBuilder {
 
     public JavadocGroupBuilder(String path, String artifactType, String title,
                                Set<String> artifactIDs = null,
-                               String pluginLocation = "https://repo.jenkins-ci.org/releases/"
+                               String pluginLocation = "https://repo.jenkins-ci.org/public/"
```

Comment:
Currently published via OSSRH, so `releases` only picks up the last _security_ release!

Reply:
Unsure; this might be weird in cases where Hudson stuff is on repo1.

Reply:
Forked Hudson artifacts use their own `groupId`s AFAICT, like https://search.maven.org/artifact/org.eclipse.hudson.stapler/stapler-core/3.0.6/jar. If they really used the same GA _and_ had newer `version`s than current Jenkins stuff, we would see problems e.g. from Dependabot all the time.

Reply:
Note that if we merge https://github.com/stapler/stapler/pull/218 and proceed with releases from there, this could be reverted if desired.